generators/app/generate.php

```diff
@@ -27,6 +27,9 @@ function generate() {
     }
 
     $this->recurse_copy(dirname(__FILE__) . "/template",$this->target_dir);
+    if(!isset($this->options->ci)) {
+        $this->recurse_rm($this->target_dir . '/.gitlab-ci.yml');
```

Comment:
I wonder if we should the file an option and then people can pass in say a URL to their default one.